src/WebJobs.Script.WebHost/Diagnostics/SystemTraceWriter.cs

```diff
@@ -53,6 +54,11 @@ public override void Trace(TraceEvent traceEvent)
                     functionName = value.ToString();
                 }
 
+                if (traceEvent.Properties.TryGetValue(ScriptConstants.TracePropertyHostVersionKey, out value) && value != null)
```

Comment:
Any reason why this one is coming from event properties as opposed to ScriptHost.Version?